db/models/user.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
-    name: DataTypes.STRING,
+    userName: DataTypes.STRING,
     email: DataTypes.STRING,
-    password: DataTypes.STRING
+    password: DataTypes.BYTEA
   }, {});
   User.associate = function(models) {
-    // associations can be defined here
+    User.belongsToMany(models.Book, {through:models.UserBook})
+    User.belongsToMany(models.User, {as:'friendId', through:models.Friend})
```

Comment:
I think your `as` here should not be `"friendId"` but maybe like `"friends"`?

Reply:
Yeah, I was unclear on this one, we have this value on the table as friendId. Will that automatically recognize this as reference or should we change that reference on the table?

Reply:
WITHOUT the usage of aliasing (i.e. `as`), this relationship will allow you to do:

```js
const user = User.findbyPk(1, { include: { model: User } });
console.log(user.users);
```

WITH aliasing set-up on your model, you can do something like this:

```js
WITHOUT the usage of aliasing (i.e. `as`), this relationship will allow you to do:

```js
// "as" key here HAS TO BE the same as what you've configure on your model `associate`
const user = User.findbyPk(1, { include: { model: User, as: "friends" } }); 
console.log(user.friends);
```